runtime/compiler/optimizer/DataAccessAccelerator.cpp

```diff
@@ -94,7 +94,13 @@ int32_t TR_DataAccessAccelerator::perform()
        !comp()->getOption(TR_MimicInterpreterFrameShape) &&
 
        // We cannot handle arraylets because hardware intrinsics act on contiguous memory
-       !comp()->generateArraylets()&& !TR::Compiler->om.useHybridArraylets())
+       !comp()->generateArraylets()&& !TR::Compiler->om.useHybridArraylets() &&
+
+       /* Disable DAA optimization for non-vectorized paths to unblock off-heap
```

Comment:
This is really a temporary change - So move this block of comment before if statement and write a TODO. Please open up OpenJ9 issue for that and reference it here in TODO.